ppr-ui/src/composables/mhrRegistration/useMhrValidations.ts

```diff
@@ -44,29 +44,45 @@ export const useMhrValidations = (validationState: any) => {
   }
 
   /** Scroll to first SECTION tag that is invalid in specified flag block. */
-  const scrollToInvalid =
-    async (flagSection: MhrSectVal, viewId: string, optionalFlags: Array<boolean> = null): Promise<boolean> => {
-      // Create an array of the _ordered_ validation flags
-      const flagBlockArr = Object.keys(validationState[flagSection].value)
-        .map(key => validationState[flagSection].value[key])
+  const scrollToInvalid = (flagSection: MhrSectVal, viewId: string): boolean => {
+    // Create an array of the _ordered_ validation flags
+    const flagBlockArr = Object.keys(validationState[flagSection].value)
+      .map(key => validationState[flagSection].value[key])
 
-      // Prepend optional flags to array (ie for review pages)
-      if (optionalFlags) flagBlockArr.unshift(...optionalFlags)
+    // Find the _first_ corresponding Section that is invalid in the specified view
+    const view = document.getElementById(viewId)
+    const invalidComponent = view.getElementsByTagName('section')[flagBlockArr.indexOf(false)]
 
-      // Find the _first_ corresponding Section that is invalid in the specified view
-      const view = document.getElementById(viewId)
-      const invalidComponent = view.getElementsByTagName('section')[flagBlockArr.indexOf(false)]
+    // If there is an invalid component, scroll to it
+    if (invalidComponent) {
+      setTimeout(() => {
+        invalidComponent.scrollIntoView({ block: 'start', inline: 'nearest', behavior: 'smooth' })
+      }, 500)
+      return false
+    }
+    return true
+  }
+
+  /** Scroll to first HEADER of invalid step in the view.
+   *  If all steps are valid scroll to first Section that is invalid
+   *  in the specified section area.
+   */
+  const scrollToInvalidReviewConfirm = (stepsValidation: Array<boolean>): boolean => {
+    // Find the _first_ corresponding step that is invalid in the specified view
+    const view = document.getElementById('mhr-review-confirm')
+    const invalidStep = view.getElementsByTagName('header')[stepsValidation.indexOf(false)]
```

Comment:
Should probably conditionally check the `view`: `view?.getElementsByTagName...`

Reply:
Good call thanks!